### PR TITLE
P0 avatar badge

### PR DESCRIPTION
**Pull request by Omkar-Ghate**
This is the avatar and badge component. I haven't made the top navbar responsive yet. Please review the design and code.

### CODE REVIEW

**Review by varun13169 — APPROVED** [review]
Looks good to me

**Review by zoef-yt — APPROVED** [review]
Clean code. Good to merge.

File: pages/components/css/buttons.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    width: 50px;
+    height: 50px;
+    color: var(--sec-dark-purple);
+    border: 1px solid var(--sec-dark-purple);
+    border-radius: 50%;
+
+}
+
+
+
```
Comment (Muzammil-khan710) [inline pages/components/css/buttons.css]: Remove extra space from here. Else all good.

**Review by sudipkundu999 — APPROVED** [review]
UI looks good!
Try to stay consistent with units.
Like, px for <10px and rem for >10px

**Review by developer-anand — APPROVED** [review]
Good work man

File: pages/components/css/buttons.css (hunk 1 above)
Comment (Omkar-Ghate) [reply pages/components/css/buttons.css]: Sure, thanks

**Comment by Omkar-Ghate** [comment]
> UI looks good! Try to stay consistent with units. Like, px for <10px and rem for >10px

Sure, I'll take care of that. Thank you!

**Pull request merged**